src/controller/event/handler/auditlog/auditlog.go

```diff
@@ -45,7 +45,7 @@ func (h *Handler) Handle(ctx context.Context, value interface{}) error {
 	switch v := value.(type) {
 	case *event.PushArtifactEvent, *event.DeleteArtifactEvent,
 		*event.DeleteRepositoryEvent, *event.CreateProjectEvent, *event.DeleteProjectEvent,
-		*event.DeleteTagEvent, *event.CreateTagEvent:
+		*event.DeleteTagEvent, *event.CreateTagEvent, *event.CreateRobotEvent, *event.DeleteRobotEvent:
```

Comment:
for consistency, should this `*event.CreateRobotEvent, *event.DeleteRobotEvent` be a new line?